sdcm/nemesis.py

```diff
@@ -419,6 +450,28 @@ def disrupt(self):
         self.log.info('Upgrade Nemesis end')
 
 
+class UpgradeNemesisOneNode(UpgradeNemesis):
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.log.info('UpgradeNemesisOneNode begin')
+        # get the number of nodes
+        # l = len(self.cluster.nodes)
+        # prepare an array containing the indexes
+        # indexes = [x for x in range(l)]
+        # shuffle it so we will upgrade the nodes in a
+        # random order
+        # random.shuffle(indexes)
+
+        # upgrade all the nodes in random order
+        # for i in indexes:
+        #     node = self.cluster.nodes[i]
+        #     self.upgrade_node(node)
+        self.upgrade_node(self.cluster.node_to_upgrade)
```

Comment:
no, self.node_to_upgrade is a node that was selected for upgrade https://github.com/scylladb/scylla-cluster-tests/blob/master/upgrade_test.py#L655 . I node per nemesis